src/modules/Providers/actions/ProviderActionsDropdown.tsx

```diff
@@ -40,48 +43,34 @@ const ProviderActionsKebabDropdown_: FC<ProviderActionsDropdownProps> = ({ data,
 
   // Returning the Dropdown component from PatternFly library
   return (
-    <Dropdown
-      className={isKebab ? undefined : 'forklift-dropdown pf-c-menu-toggle'}
-      isOpen={isOpen}
-      onOpenChange={setIsOpen}
-      onSelect={onSelect}
-      toggle={(toggleRef: Ref<MenuToggleElement>) => (
-        <MenuToggle
-          ref={toggleRef}
-          onClick={onToggleClick}
-          isExpanded={isOpen}
-          variant={isKebab ? 'plain' : 'default'}
-        >
-          {isKebab ? <EllipsisVIcon /> : t('Actions')}
-        </MenuToggle>
-      )}
-      shouldFocusToggleOnSelect
-      popperProps={{
-        position: 'right',
-      }}
-    >
-      <DropdownList>{ProviderActionsDropdownItems({ data })}</DropdownList>
-    </Dropdown>
+    <ModalHOC>
```

Comment:
There's some extra space at the bottom:

<img width="217" alt="Screenshot 2025-04-10 at 1 15 08 PM" src="https://github.com/user-attachments/assets/0a087428-3c05-4f1e-8666-a4043c317e73" />

Reply:
looking at it closer, looks like a `,` is rendered at the end here

Reply:
nice, fixed thanks!